src/NUnitTestAdapter/Properties/AssemblyInfo.cs

```diff
@@ -21,5 +21,5 @@
 [assembly: ComVisible(false)]
 
 [assembly: Guid("c0aad5e4-b486-49bc-b3e8-31e01be6fefe")]
-[assembly: AssemblyVersion("1.2.0.1")]
-[assembly: AssemblyFileVersion("1.2.0.1")]
+[assembly: AssemblyVersion("1.9.9.0")]
```

Comment:
A possible idea for the future would be to set up an entirely separate
package for development versions.

On Tue, Mar 31, 2015 at 9:12 AM, Terje Sandstrom <notifications@github.com>
wrote:

> In src/NUnitTestAdapter/Properties/AssemblyInfo.cs
> <https://github.com/nunit/nunit-vs-adapter/pull/66#discussion_r27494465>:
>
> > @@ -21,5 +21,5 @@
> >  [assembly: ComVisible(false)]
> >
> >  [assembly: Guid("c0aad5e4-b486-49bc-b3e8-31e01be6fefe")]
> > -[assembly: AssemblyVersion("1.2.0.1")]
> > -[assembly: AssemblyFileVersion("1.2.0.1")]
> > +[assembly: AssemblyVersion("1.9.9.0")]
>
> Yes it should.
> I just kept it at 1.9.9 while beta'ing it, as vsix'es dont have any beta
> concept like nuget packages has.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit-vs-adapter/pull/66/files#r27494465>.
>